### DISCUSSION TITLE
From which Yi firmware version is it best to install Yi-Hack?

### DISCUSSION THREAD

**Discussion by chrbar11** [roleoroleo/yi-hack-Allwinner-v2#957] [General]
Hi,

I have Yi 1080p Home and Yi Outdoor 1080p cameras.
I've installed Yi-Hack from the native Yi firmware version 9.0.*.

Do you recommend updating Yi firmware (to 12.0.*) via the Yi app before installing Yi-Hack?
Does this bring new features, better performance, bug fixes or other benefits?

Thanks a lot for your help and advice :)
Chris

**Comment by roleoroleo** [comment]
> Does this bring new features, better performance, bug fixes or other benefits?

It depends on the cam. The main feature added in v12 is the cloud AI detection.
If you don't have problems with 9.0.x probably the update is not necessary.

If you want to upgrade, follow this procedure:
- Remove the sd card and reboot the cam.
- Update the cam from the app.
- Shutdown the cam.
- Insert SD card and reboot.
